.github/workflows/beekeeper.yml

```diff
@@ -46,7 +46,7 @@ jobs:
       - name: Prepare local cluster
         run: |
           printf ${{ secrets.CR_PAT }} | docker login ghcr.io -u bee-worker --password-stdin
-          make beekeeper BEEKEEPER_INSTALL_DIR=/usr/local/bin BEEKEEPER_USE_SUDO=true
+          make beekeeper BEEKEEPER_INSTALL_DIR=/usr/local/bin BEEKEEPER_USE_SUDO=true BEEKEEPER_BRANCH=bigint-serialization
```

Comment:
Will this stay as `BEEKEEPER_BRANCH=bigint-serialization` or just something temporary?

Reply:
this needs to be reverted

Reply:
done